features-json/css-anchored-container-queries.json

```diff
@@ -0,0 +1,720 @@
+{
+  "title":"CSS Anchored Container Queries",
+  "description":"Anchored container queries allow querying which position-try-fallbacks option was applied to an anchor-positioned element, enabling conditional styling based on the resolved anchor position.",
+  "spec":"https://drafts.csswg.org/css-anchor-position-2/#anchored-container-queries",
+  "status":"wd",
+  "links":[
+    {
+      "url":"https://developer.chrome.com/blog/anchored-container-queries",
+      "title":"Chrome Developers blog post"
+    },
+    {
+      "url":"https://drafts.csswg.org/css-anchor-position-2/",
+      "title":"CSS Anchor Positioning Level 2 specification"
+    },
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/CSS/Guides/Containment/Container_queries",
+      "title":"MDN: CSS container queries"
+    },
+    {
+      "url":"https://codepen.io/web-dot-dev/pen/jEWKRRv",
+      "title":"Demo: Anchored container queries for flipping tooltip with arrow"
```

Comment:
We should add a note: starting from Chrome 144, due to the implementation of `scrollable containing-block`, the elements must be set to `position: fixed`.

https://issues.chromium.org/issues/438515315#comment32